packages/itwinui-css/src/utils/component-action/link-box.scss

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../mixins';
+
+@mixin iui-link-box {
+  position: relative;
+  width: fit-content;
+  height: fit-content;
+  cursor: pointer;
```

Comment:
`cursor: pointer` should go on link-overlay.

`width:fit-content` and `height: fit-content` shouldn't be necessary. This class should have no opinion on the sizing. but i think you added this because `div`s automatically take up the available width. This can be fixed using the `as` prop.
```
<LinkBox as={Tile}>
```
```
<div class='iui-tile iui-link-box'>
```

however, in #1110, I've also added `position: relative` to the Tile, so maybe there is no need to even wrap the Tile in Linkbox.

Reply:
I'm not wrapping Tile in LinkBox. 

Tested with MenuItem, should work:
```js
<LinkBox as={MenuItem} key='2'>
  <LinkOverlay as='button' onClick={}>Item #2</LinkOverlay>
</LinkBox>
```